t/job-manager/plugins/jobspec-update-job-list.c

```diff
@@ -71,6 +78,7 @@ static int sched_cb (flux_plugin_t *p,
                                      "update failure");
         return -1;
     }
+    updated = true;
     return 0;
 }
```

Comment:
For these tests it is probably fine to post updates from `job.validate` since they are run in a controlled environment (i.e. no out-of-band updates are coming in via the update service). Updates are only validated when posted through the update service...